core/job/service/job_service.go

```diff
@@ -757,6 +760,12 @@ func (j *JobService) bulkAdd(ctx context.Context, tenantWithDetails *tenant.With
 		raiseJobEventMetric(tenantWithDetails.ToTenant(), job.MetricJobEventStateAdded, len(addedJobs))
 	}
 
+	for _, addedJob := range addedJobs {
+		if addedJob.Spec().Schedule().CatchUp() {
+			j.logger.Warn("catchup for job %s is enabled, starting from %s", addedJob.GetName(), addedJob.Spec().Schedule().StartDate().String())
```

Comment:
The log here is only for server, let's put for the client as well. Also, if there's no particular reason to add the start date, better remove it.